lib/ansible/plugins/filter/core.py

```diff
@@ -271,13 +271,17 @@ def get_encrypted_password(password, hashtype='sha512', salt=None):
     # TODO: find a way to construct dynamically from system
     cryptmethod = {
         'md5': '1',
-        'blowfish': '2a',
+        'blowfish': '2b',
```

Comment:
2a  | Blowfish (not in mainline glibc; added in some
                  | Linux distributions)

from the glibc man page for crypt.